options/options.go

```diff
@@ -240,3 +247,21 @@ func validateFormat(format string) bool {
 		return false
 	}
 }
+
+// isCheckValidForLocalRepo returns an error if the check is not valid for local repository.
+func (o *Options) isCheckValidForLocalRepo() error {
```

Comment:
Each check self-declares a supported request type - https://github.com/ossf/scorecard/blob/main/checks/license.go#L39. Can use this to find whether the check is supported for Local or not. Here is an example - https://github.com/ossf/scorecard/blob/main/policy/policy.go#L211

Reply:
Took care of it. Thanks